util/customTransformer.js

```diff
@@ -233,7 +235,28 @@ async function userTransformHandler(
       });
 
       let userTransformedEvents = [];
-      if (res.codeVersion && res.codeVersion === "1") {
+      if (res.language && res.language !== "javascript") {
```

Comment:
It's better to check for "python" rather than the negation. Allows for supporting a different language later.